generators/client/templates/angular/src/main/webapp/app/admin/configuration/_configuration.service.ts

```diff
@@ -27,7 +30,7 @@ export class <%=jhiPrefixCapitalized%>ConfigurationService {
     }
 
     get(): Observable<any> {
-        return this.http.get('management/configprops').map((res: Response) => {
+        return this.http.get(<% if (authenticationType === 'uaa') { %>'<%= uaaBaseName.toLowerCase() %>/<% } else { %>SERVER_API_URL + '<% } %>management/configprops').map((res: Response) => {
```

Comment:
Wondering whether we could define a function or a variable to make it more readable by replacing: 

[<% if (authenticationType === 'uaa') { %>'<%= uaaBaseName.toLowerCase() %>/<% } else { %>SERVER_API_URL + '<% } %>](url)

Reply:
yes, this is now in too many places, we should just use a new variable called apiUrl and do this logic on the generator code and just use the variable in templates. @thomastardy i'll do it on master and then you can rebase to master to just use the template

Reply:
@thomastardy I have refactored this into 2 variables so you can use `apiUrlPrefix` variable
Check this https://github.com/jhipster/generator-jhipster/pull/6824 I'll merge it into master once tests are green and then you can rebase

Reply:
@gmarziou very good point, thanks for the feedback.
@deepu105 I will wait till #6824 gets merged. Or if you which you can do these five small changes in #6824 as well and close this PR. I leave this up to you.